server/isomorphic-routing/test/index.js

```diff
@@ -0,0 +1,81 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { getCacheKey } from '..';
+
+jest.mock( 'redux-form/es/reducer', () => require( 'lodash' ).identity );
```

Comment:
This module has es6 imports which was breaking tests. This allows the tests to run although it's likely a symptom of some lacking configuration.

Improving configs is outside the scope of this PR.